highlighters/main/main-highlighter.zsh

```diff
@@ -76,7 +76,7 @@ _zsh_highlight_main_add_region_highlight() {
   integer start=$1 end=$2
   shift 2
 
-  if (( in_alias )); then
+  if (( $#in_alias )); then
```

Comment:
Does `(( ${+in_alias[1]} ))` perform measurably better?

Reply:
Oddly enough it preforms measurably worse.
```
a=(); time ( repeat 100000; do (( ${+a[1]} )); done; )
```
array | `$#a` | `${+a[1]}`
-- | -- | --
`a=()` | 0.16 | 0.29
`a=(1 2 3)` | 0.17 | 0.32

Reply:
Curious, yes.  Agree that no change is needed in this case, but since we use `${+a[1]}` elsewhere, it's worth noting that that idiom _is_ faster for large arrays.

Reply:
Thanks for testing!